### ISSUE TITLE
Platters are disappearing for some time when 3D Touched/Long Pressed

### ISSUE THREAD

**Issue by zyan910** [ioscreatix/Maize#160]
iOS 10.1.1, iPhone 6
https://streamable.com/4793u

**Comment by andrewwiik** [comment]
This has always been a "can't fix" because the view needs to be removed and readded to the hierarchy, maybe someone who does regular iOS Dev knows what I'm doing wrong in my transition but even apple has this problem when closing a HomeKit module as you will be able to see if you disable Maize (still working on the homekit module)

**Comment by zyan910** [comment]
If i remove all the toggles and keep JUST THE CONNECTIVITY AND THE MUSIC toggle, then this error is not there, or even if i add ALL THE TOGGLES, the error isn't there, i just use 16 toggles and the error persists

**Comment by andrewwiik** [comment]
I believe you are wrong as I have this "bug" even when having only 4 modules

**Comment by zyan910** [comment]
No, I've tried it myself...I was playing around with Maize's setting to see if it looked cool with just the connectivity platter, and when i long pressed it, this bug wasn't there, the animation was perfect
